core/module_test.cc

```diff
@@ -295,14 +292,13 @@ TEST_F(ModuleTester, GenerateTCGraph) {
    *         \
    *          t4
    */
-  Module *t1, *t2, *t3, *t4, *m1, *m2, *m3;
-  EXPECT_EQ(0, create_acme("m1", &m1));
-  EXPECT_EQ(0, create_acme("m2", &m2));
-  EXPECT_EQ(0, create_acme("m3", &m3));
-  EXPECT_EQ(0, create_acme_with_task("t1", &t1));
-  EXPECT_EQ(0, create_acme_with_task("t2", &t2));
-  EXPECT_EQ(0, create_acme_with_task("t3", &t3));
-  EXPECT_EQ(0, create_acme_with_task("t4", &t4));
+  EXPECT_NE(nullptr, m1 = create_acme("m1", &perr));
```

Comment:
Existing code had `EXPECT_EQ` here, `ASSERT_*` should be used for null pointer check.